gateway/src/apicast/policy/ngx_variable.lua

```diff
@@ -17,7 +17,8 @@ local function context_values()
 end
 
 function _M.available_context(policies_context)
-  return LinkedList.readonly(context_values(), policies_context)
+  return LinkedList.readwrite({}, LinkedList.readonly(
```

Comment:
Should we make it read-write in the `LiquidTemplateString:render` function instead?
That way other callers of this function will get a read-only list which I think is less surprising.